Student/Student/StudentAppDelegate.swift

```diff
@@ -55,10 +55,11 @@ class StudentAppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDele
         try? AVAudioSession.sharedInstance().setCategory(.playback, mode: .default)
 
         if launchOptions?[.sourceApplication] as? String == Bundle.teacherBundleID,
-            let url = launchOptions?[.url] as? URL,
-            let components = URLComponents(url: url, resolvingAgainstBaseURL: true),
-            components.path == "student_view",
-            let fakeStudent = LoginSession.mostRecent(in: .shared, forKey: .fakeStudents) {
+           let url = launchOptions?[.url] as? URL,
```

Comment:
identation seems a bit off here

Reply:
it was, now it's the correct indentation, at least for Xcode 13.